.gitignore

```diff
@@ -2,6 +2,7 @@ Makefile
 Makefile.in
 README.html
 aclocal.m4
+ar-lib
```

Comment:
What is this file, and should it be included in the source tarball?

Reply:
Does `make dist` include `ar-lib` in the source tarball automatically? I'm guessing yes.

Reply:
`ar-lib` says it's a `# Wrapper for Microsoft lib.exe`. My guess it;s a way to provide an `ar`-like interface on MSVC systems.

It should gt into tarball automatically (unless `abyss` does some unusual overrides). I'll double-check.

Reply:
Tested `make dist` today. `ar-lib` gets into tarball as expected.